src/components/RadioGroup.stories.tsx

```diff
@@ -8,6 +8,14 @@ import componentDecorators from '../../.storybook/component-decorators'
 export default {
   title: 'Components/Radio Groups',
   component: RadioGroup,
+  parameters: {
+    a11y: {
+      /*
+    Disabling "nested-interactive" rule until there's a fix from Ionic: https://github.com/ionic-team/ionic-framework/issues/23511
+     */
```

Comment:
Weird comment formatting... lol
```suggestion
      // Disabling "nested-interactive" rule until there's a fix from Ionic: https://github.com/ionic-team/ionic-framework/issues/23511
```